src/TestLogger/Extensions/NUnitTestAdapter.cs

```diff
@@ -21,7 +21,7 @@ public List<TestResultInfo> TransformResults(List<TestResultInfo> results, List<
                 // is passed as a trait in the test platform. NUnit explicit attribute spec:
                 // https://docs.nunit.org/articles/nunit/writing-tests/attributes/explicit.html
                 if (result.Outcome == Microsoft.VisualStudio.TestPlatform.ObjectModel.TestOutcome.None &&
-                    result.TestCase.Traits.Any(trait => trait.Name.Equals(ExplicitLabel, StringComparison.OrdinalIgnoreCase)))
+                    result.Traits.Any(trait => trait.Name.Equals(ExplicitLabel, StringComparison.OrdinalIgnoreCase)))

```

Comment:
There might be a catch here. 

Traits are a property of the test discovery (adapters use reflection to find the traits when they find the test cases). Not all adapters _copy_ the traits to the corresponding `TestResult` object. Using `TestResult.TestCase` may be safer bet.

Reply:
Got it, see a4ed98580651736c385cd2eda72efcb016caf834. The change wouldn't happen in this adapter.